docs/core/deploy.md

```diff
@@ -34,11 +41,13 @@ To be able to use the autocomplete feature of the point inputs you have to:
 
 ## Worldwide Setup
 
-GraphHopper is able to handle coverage for the whole [OpenStreetMap road network](http://planet.osm.org/).
-It needs approximately 25GB RAM for the import (CAR only) and ~1 hour (plus ~5h for contraction).
-If you can accept slower import times this can be reduced to 14GB RAM - you'll need to set `datareader.dataaccess=MMAP` in the config file.
+GraphHopper is able to handle coverage for the world-wide [OpenStreetMap road network](http://planet.osm.org/).
+
+Without enabled CH the import step alone requires ~60GB RAM and takes ~3h for the import. If you can accept
```

Comment:
Parsing the OSM planet file and creating the GraphHopper base graph requires ~60GB RAM ...

Running the CH preparation, which is required for maximum routing speed, takes ... without turn costs and ... with turn costs.

Running the LM preparation ... takes ...

It is also possible to add CH/LM preparations for existing profiles after the initial import. Adding or modifying profiles is not possible and you need to run a new import instead.